action.yml

```diff
@@ -3,13 +3,25 @@ description: Updates README with the recent GitHub activity of a user
 author: jamesgeorge007
 
 inputs:
+  COMMIT_MSG:
+    description: 'Commit message used while committing to the repo'
+    default: ':zap: Update README with the recent activity'
+    required: false
   GH_USERNAME:
     description: 'Your GitHub username'
     default: ${{ github.repository_owner }}
     required: false
-  COMMIT_MSG:
-    description: "Commit message used while committing to the repo"
-    default: ":zap: Update README with the recent activity"
+  LINES:
+    description: The number of lines to generate
+    default: '5'
+    required: false
+  NO_COMMIT:
```

Comment:
Can you explain the motivation behind introducing this input param?

Reply:
Which one? `NO_COMMIT` or `LINES`?
- `NO_COMMIT`: if someone wants to do some other edit to the README they can do it and then push all the changes in a single commit.
- `LINES`: they can select the number of lines from the workflow

Reply:
We're checking to see if there is any difference between the content enclosed within the respective comments and the latest data being pulled. Any change made to other sections of the `README` doesn't affect this.

Reply:
@jamesgeorge007 That's not the issue: I may want to do some other changes to the README and then commit everything together, instead of having multiple commits.